### PR TITLE
chore: document endpoints for files and projects

### PR DESCRIPTION
**Pull request by wollefitz**
Closes https://github.com/camunda/cawemo/issues/11294

Should be backported to all docs later or equal than `1.9`.

### CODE REVIEW

File: content/reference/rest-api/files/get.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  </tr>
+  <tr>
+    <td>content</td>
+    <td>string</td>
+    <td>Content of the requested file.</td>
+  </tr>
+  <tr>
+    <td>metadata</td>
+    <td>object</td>
+    <td>Metadata of the file, see [File resource]({{< ref "/reference/rest-api/files/_index.md#resource-representation" >}}).</td>
```
Comment (jfriedenstab) [inline content/reference/rest-api/files/get.md:60]: The link is not rendered correctly (but I'm not sure if this is only a problem in the local watch mode):

grafik

**Review by jfriedenstab — CHANGES_REQUESTED** [review]
Looks good 👍🏻; just one problem with a link.

File: content/reference/rest-api/files/get.md (hunk 1 above)
Comment (wollefitz) [reply content/reference/rest-api/files/get.md]: Fixed. Markdown within HTML tables can only be rendered if its on its on new line - I replaced it with a HTML link now.

**Review by jfriedenstab — APPROVED** [review]
🚀

**Pull request merged**